service.subtitles.traduttorianonimi/service.py

```diff
@@ -58,84 +60,41 @@
 language = addon.getLocalizedString
 main_url = "http://traduttorianonimi.it"
 
-NotifyLogo = os.path.join(cwd, "likelogo.png")
+NotifyLogo = xbmc.translatePath(os.path.join(cwd, "resources", "likelogo.png")).decode("utf-8")
 
 if not os.path.isfile(NotifyLogo):
     NotifyLogo = None
 
-HEADERS = {"user-agent": "Kodi-SubtitleService-TraduttoriAnonimi"}
-
-
 def notify(message, header=scriptname, icon=NotifyLogo, time=5000):
     dialog.notification(header, message, icon=icon, time=time)
 
-
 class LogStream(object):
     def write(self, data):
         xbmc.log("*** [service.subtiles.traduttorianonimi] -> {0}".format(data.encode('utf-8', "ignore")),
                  level=xbmc.LOGNOTICE)
 
-
-log = logging.getLogger("TraduttoriAnonimi")
-log.setLevel(logging.DEBUG)
-style = logging.Formatter("{%(levelname)s} %(name)s.%(funcName)s() -->> %(message)s")
-consoleHandler = logging.StreamHandler(LogStream())
-consoleHandler.setFormatter(style)
-log.addHandler(consoleHandler)
-
-
-def magicUnicode(data):
-    """
-    Return Unicode (utf-8) text encoded or exactly the input data (if already encoded)
-    :param data: data (text) to encode
-    :return:
-    """
-    # unicode is not defined in python3.x
-    if type(data) != bytes:
-        return str(data).encode("utf-8")
-    else:
-        return data
-
-
-def retriveURL(url, headers=HEADERS):
-    try:
-        log.debug("GET Request => HEADERS={0} ; URL={1}".format(headers, url))
-        q = requests.get(url, headers=headers)
-        log.debug("GET Request <= Response HEADERS={0}".format(q.headers))
-        return q
-    except:
-        log.error("An Error is occurred", exc_info=True)
-        notify(language(30001), time=3000)  # Network Error. Check your Connection
-        return None
-
-def overrideRetriveURL(self, url, headers=HEADERS):
-    return retriveURL(url, headers)
-
-def ovverideLogging(self):
-    self.log = log
-
+logging.basicConfig(
```

Comment:
Please use xbmc.log. You can extend the python logger to use `xbmc.log` by implementing something like this:

https://github.com/xbmc/generator-kodi-addon/blob/master/generators/app/templates/resources/lib/kodilogging.py